src/modules/Classes.js

```diff
@@ -0,0 +1,45 @@
+export default class Books {
+  constructor(books) {
+    this.books = books;
+  }
+
+  get() {
+    return this.books;
+  }
+
+  // method to add book by geting data from dom
+  addBook(e) {
+    e.preventDefault();
+    const title = document.getElementById('title').value;
+    const author = document.getElementById('author').value;
+    this.books.push({ title, author });
+    localStorage.setItem('books', JSON.stringify(this.books));
+    window.location.reload();
+  }
+  // method to create books and append to dom
+
+  createBookFromLocal() {
+    for (let i = 0; i < this.books.length; i += 1) {
+      const booksShow = document.querySelector('.books-show');
+      booksShow.style.display = 'block';
+      const book = document.createElement('li');
+      const p = document.createElement('p');
+      p.innerHTML = `${this.books[i].title} by ${this.books[i].author}`;
+      book.append(p);
+      const remove = document.createElement('button');
+      remove.innerText = 'remove';
+      remove.addEventListener('click', () => {
+        this.remove(this.books[i].title);
+      });
+      book.append(remove);
+      booksShow.appendChild(book);
+    }
```

Comment:
- Nice, The App is working as expected 💯 
- 
![Capture](https://user-images.githubusercontent.com/63972670/197772075-66234478-a2ce-4e1e-b6a6-16102457531c.PNG)